Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm

```diff
@@ -2566,7 +2566,7 @@ - (void)scrollViewDidZoom:(UIScrollView *)scrollView
     [self _scheduleVisibleContentRectUpdateAfterScrollInView:scrollView];
 
 #if ENABLE(MODEL_PROCESS)
-    [self _setWebViewTransform3DForModel:[scrollView zoomScale]];
+    [_contentView _setWebViewTransform3DForModel:[scrollView zoomScale]];
```

Comment:
we should rename this since we're not setting the transform on the WebView anymore